### DISCUSSION TITLE
Add a way to add authentication headers

### DISCUSSION THREAD

**Discussion by SuperMeepEnby** [spacecowboy/Feeder#194] [Ideas]
Currently we can not add RSS which needs to authenticate. There is a way to get elements of RSS feeds by adding authentication headers. This great feature would allow people to read their paid newspapers or any website behind an authentication.

**Comment by spacecowboy** [comment]
you can't add headers, that's true.

but you can add URLs which contain usernames and passwords,  like

https://username:password@example.com/rss.xml

these are converted into basic auth headers
